src/main/java/TaskManager.java

```diff
@@ -0,0 +1,86 @@
+import java.util.*;
+
+public class TaskManager {
+    protected static Task[] tasks = new Task[100];
+    protected static int inputAmount = 0;
+
+    public static void addTask(String reply){
+        String choice = reply.split(" ")[0];
+        String newTask = reply.replace(choice+" ", "");
+        switch(choice.toUpperCase()) {
+            case "TODO":
+                ToDo todo = new ToDo(newTask);
```

Comment:
the indent
```suggestion
        switch(choice.toUpperCase()) {
        case "TODO":
            ToDo todo = new ToDo(newTask);
```

Reply:
there should not be indentation for switch-case statements

Reply:
Thank you for your suggestion! I will fix this indentation later!